### PR TITLE
update CHANGELOG to v11.0.0

### PR DESCRIPTION
**Pull request by smbea**
**Context:** release new popup menu to be integrated into bpmn-js

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,10 +6,16 @@ All notable changes to [diagram-js](https://github.com/bpmn-io/diagram-js) are d
...
+## 11.0.0
 * `FEAT`: new popupMenu UI featuring menu and group titles, search, entry descriptions and documentation urls ([#687](https://github.com/bpmn-io/diagram-js/pull/687))
 * `FEAT`: introduction of `.djs-parent` class to canvas and popup menu root ([#687](https://github.com/bpmn-io/diagram-js/pull/687))
+* `CHORE`: provide UI utilities through `diagram-js/lib/ui` ([#692](https://github.com/bpmn-io/diagram-js/pull/692))
+* `DOCS`: update popup menu provider example ([#678](https://github.com/bpmn-io/diagram-js/pull/678))
+* `DEPS`: bump utility dependencies
 
+### Breaking Changes
+* Improved popup menu UI and behaviour, resulting in the modification of some of the composing HTML elements and CSS classes.
+* The popup menu now exists outside of the canvas container. The `djs-parent` class was introduced to the canvas and popup menu roots.
```
Comment (barmac) [inline CHANGELOG.md:18]: Let's answer here two questions:
1) What is breaking?
2) How do I migrate to new version?

**Comment by nikku** [comment]
Let us release a pre-release first to incorporate in bpmn-js. 

I'd love to get https://github.com/bpmn-io/diagram-js/pull/681 incorporated, too, which may be a breaking change (depending on how we spin it).

**Comment by nikku** [comment]
Yes, cf. https://www.npmjs.com/package/diagram-js?activeTab=versions.

To be published on the `next` tag so users don't accidentally consume it.

**Comment by barmac** [comment]
> Let us release a pre-release first to incorporate in bpmn-js.
> 
> I'd love to get #681 incorporated, too, which may be a breaking change (depending on how we spin it).

I'll look into this as the contributor seems to be not active anymore.

**Comment by barmac** [comment]
> > Let us release a pre-release first to incorporate in bpmn-js.
> > I'd love to get #681 incorporated, too, which may be a breaking change (depending on how we spin it).
> 
> I'll look into this as the contributor seems to be not active anymore.

It's merged now.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,15 +6,22 @@ All notable changes to [diagram-js](https://github.com/bpmn-io/diagram-js) are d
 
 _**Note:** Yet to be released changes appear here._
 
+## 11.0.0
+
 * `FEAT`: drop KeyboardEvent.keyCode in favor of `code` ([#681](https://github.com/bpmn-io/diagram-js/pull/681))
+* `FEAT`: introduction of `.djs-parent` class to canvas and popup menu root ([#687](https://github.com/bpmn-io/diagram-js/pull/687))
+* `FEAT`: new popupMenu UI featuring menu and group titles, search, entry descriptions and documentation urls ([#687](https://github.com/bpmn-io/diagram-js/pull/687), [#696](https://github.com/bpmn-io/diagram-js/pull/696), [#697](https://github.com/bpmn-io/diagram-js/pull/697), [#699](https://github.com/bpmn-io/diagram-js/pull/699))
```
[the remark this answers is not in the document]
Comment (barmac) [reply CHANGELOG.md:13]: Usually we link the issue if it's in the same repo. In this case, you could link https://github.com/bpmn-io/diagram-js/issues/686

**Review by barmac — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,15 +6,22 @@ All notable changes to [diagram-js](https://github.com/bpmn-io/diagram-js) are d
...
+* `DOCS`: correct ElementRegistry signatures ([#698](https://github.com/bpmn-io/diagram-js/pull/698))
+* `DEPS`: bump utility dependencies
 
 ### Breaking Changes
-* `FEAT`: new popupMenu UI featuring menu and group titles, search, entry descriptions and documentation urls ([#687](https://github.com/bpmn-io/diagram-js/pull/687))
-* `FEAT`: introduction of `.djs-parent` class to canvas and popup menu root ([#687](https://github.com/bpmn-io/diagram-js/pull/687))
+* HTML structure and CSS classes of the popup menu were modified. Ensure alignment with the new structure in your custom implementation.
+* There is no longer a single root (`.djs-container`). The global selector for both the canvas and popup menu roots is now `.djs-parent`.
 * Keyboard-related features no longer use KeyboardEvent#keyCode.
   Use a polyfill (e.g. [keyboardevent-key-polyfill](https://www.npmjs.com/package/keyboardevent-key-polyfill) if you need to support old browsers.
```
Comment (nikku) [inline CHANGELOG.md:23]:
```suggestion
  Use a polyfill (e.g. [keyboardevent-key-polyfill](https://www.npmjs.com/package/keyboardevent-key-polyfill)) if you need to support old browsers.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,15 +6,22 @@ All notable changes to [diagram-js](https://github.com/bpmn-io/diagram-js) are d
...
 _**Note:** Yet to be released changes appear here._
 
+## 11.0.0
+
 * `FEAT`: drop KeyboardEvent.keyCode in favor of `code` ([#681](https://github.com/bpmn-io/diagram-js/pull/681))
+* `FEAT`: introduction of `.djs-parent` class to canvas and popup menu root ([#687](https://github.com/bpmn-io/diagram-js/pull/687))
+* `FEAT`: new popupMenu UI featuring menu and group titles, search, entry descriptions and documentation urls ([#686](https://github.com/bpmn-io/diagram-js/issues/686), [#695](https://github.com/bpmn-io/diagram-js/issues/695), [#697](https://github.com/bpmn-io/diagram-js/pull/697), [#699](https://github.com/bpmn-io/diagram-js/pull/699))
+* `FEAT`: provide UI utilities through `diagram-js/lib/ui` ([#692](https://github.com/bpmn-io/diagram-js/pull/692))
+* `DOCS`: update popup menu provider example ([#678](https://github.com/bpmn-io/diagram-js/pull/678))
+* `DOCS`: correct ElementRegistry signatures ([#698](https://github.com/bpmn-io/diagram-js/pull/698))
```
Comment (nikku) [inline CHANGELOG.md:16]:
```suggestion
* `DOCS`: correct `ElementRegistry` method signatures ([#698](https://github.com/bpmn-io/diagram-js/pull/698))
```

**Pull request merged**